artemis-junit/src/test/java/org/apache/activemq/artemis/junit/MultipleEmbeddedActiveMQResourcesTest.java

```diff
@@ -38,6 +38,11 @@
    static final String ASSERT_RECEIVED_FORMAT = "Message should have been received from %s";
    static final String ASSERT_COUNT_FORMAT = "Unexpected message count in queue %s";
 
+   static {
+      ThreadLeakCheckRule.addKownThread("MemoryPoolMXBean notification dispatcher");
```

Comment:
This was already reported on [HORNETQ-1206](https://issues.jboss.org/browse/HORNETQ-1206) and they argued that these threads are not managed by broker itself. They are created by IBM JDK.
I don't think we can kill some threads but I am not sure.